src/persistence/environmental_authority.js

```diff
@@ -335,5 +335,35 @@ module.exports = (
       )
         .then(layers => layers.rows[0].collection)
     ),
+
+    /**
+     * Select biomes by a given environmental authority.
+     *
+     * @param {String} envAuthority environmental authority name to filter by
+     *
+     * @returns {Object} GeoJson Object with biomes as features from a FeatureCollection
+     */
+    findBiomeByEA: envAuthority => (
```

Comment:
This name would be better as "findBiomesLayerById" (I think ByEA is unnecessary since this is the EA persistence) and would match more with the other functions

Reply:
Done